packages/graph/migrations/.eslintrc.cjs

```diff
@@ -4,4 +4,12 @@ module.exports = {
   env: {
     node: true,
   },
+  overrides: [
+    {
+      files: ["**/migration/*.ts"],
```

Comment:
These files are autogenerated so I guess we can permanently allow `_` in their names.